pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/pidigits1.rkt

```diff
@@ -1,4 +1,6 @@
 #lang racket/base
+;; SPDX-License-Identifier: LGPL-3.0-or-later
```

Comment:
I would imagine so, given the age, but https://cvs.savannah.gnu.org/viewvc/scm/scm/pi.scm?view=markup said LGPL 3, and I didn't do any further forensics. Certainly it can be distributed under LGPL 3, so IMO it would be ok to do this now and change it to 2 or 2.1 if we later work out to our satisfaction that one of those is applicable.